examples/twittermap/web/public/javascripts/sidebar/controllers.js

```diff
@@ -127,7 +127,13 @@ angular.module("cloudberry.sidebar", ["cloudberry.common"])
           }
           else{
             sampleTweets = sampleTweets.concat(resultSet[0]);//oldest tweet will be at front
+            //Update 1 tweet immdiately 
```

Comment:
Typo in the comment "immdiately"

Reply:
Got it, wish IDLE can have spell checking feature